pwa-devdocs/_drafts/configuration-management/buildpack-cli/create-project/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: create-project
+---
+
+<!-- TODO: Publish this topic under the buildpack-cli reference directory when the feature becomes available -->
+
+The `create-project` sub-command of the [`pwa-buildpack`][] CLI is a [scaffolding][] tool used to create a fresh PWA Studio app in a new project directory.
+
+Normally this command is used indirectly through the `@magento/pwa` scaffolding tool.
+The tool collects the parameter values through its interactive prompt and passes them onto the command as options.
+
+This tool is also available for developers or third-party automation tools to use directly.
+
+**Example:** Using the command with `npx`:
+
+```sh
+npx @magento/pwa-buildpack create-project ./new-pwa \
+  --name @magezilla/new-pwa \
+  --template venia-concept \
+  --backend-url https://local.magento \
+  --author MageZilla
+```
+
+This example creates a new project in the `./new-pwa` folder.
+The package metadata for the project lists the project name as `@magezill/new-pwa` and the author as `MageZilla`.
+It also creates a `.env` file with the backend URL set to `https://local.magento`.
+
+After creating the project, it installs the package dependencies, including the Venia, Peregrine and Buildpack libraries needed to build the application.
```

Comment:
Does it? I thought it just spit out a message saying "you (the developer) should do it".

Reply:
It does if you choose that option! It's the last prompt in the questionnaire.